firebase-perf/src/main/java/com/google/firebase/perf/FirebasePerformance.java

```diff
@@ -147,6 +152,10 @@ public static FirebasePerformance getInstance() {
         RemoteConfigManager.getInstance(),
         ConfigResolver.getInstance(),
         GaugeManager.getInstance());
```

Comment:
Do we still need the this(...)?

Reply:
For now yes, I'm working on figuring out a way to get this removed as well.

Reply:
We are trying to unify to have a common constructor. Currently there are 2 separate constructors, one for testing and one for the registrar which should ideally be unified.

Reply:
Done!